lib/grp.gi

```diff
@@ -4453,9 +4453,14 @@ end);
 ##  anyhow,you should compute it before computing the maximal normal
 ##  subgroups.
 ##
+##  *Note* that for abelian and solvable groups the maximal normal subgroups
+##  can be computed very quickly. Thus if you suspect your group to be
+##  abelian or solvable, then check it before computing the maximal normal
+##  subgroups.
+##
```

Comment:
Added the note to the manual and added an infinite example to emphasize that ````MaximalNormalSubgroups```` works for some infinite groups.